README.md

```diff
@@ -1,56 +1,52 @@
+This project is a fork of [FlakeHell](https://github.com/life4/flakehell). FlakeHell and the official fork of it at
+[flakehell/flakehell](https://github.com/flakehell/flakehell) are no longer maintained and do not work with Flake8 4.0.x.
 
-It's a [Flake8](https://gitlab.com/pycqa/flake8) wrapper to make it cool.
+FlakeHeaven works with Flake8 4.0.1 or greater.
 
-+ [Lint md, rst, ipynb, and more](https://flakehell.readthedocs.io/parsers.html).
-+ [Shareable and remote configs](https://flakehell.readthedocs.io/config.html#base).
-+ [Legacy-friendly](https://flakehell.readthedocs.io/commands/baseline.html): ability to get report only about new errors.
++ [Lint md, rst, ipynb, and more](https://github.com/mcarans/flakeheaven/blob/master/docs/parsers.md).
++ [Shareable and remote configs](https://github.com/mcarans/flakeheaven/blob/master/docs/config.md#base).
++ [Legacy-friendly](https://github.com/mcarans/flakeheaven/blob/master/docs/commands/baseline.md): ability to get report only about new errors.
 + Caching for much better performance.
-+ [Use only specified plugins](https://flakehell.readthedocs.io/config.html#plugins), not everything installed.
-+ [Make output beautiful](https://flakehell.readthedocs.io/formatters.html).
++ [Use only specified plugins](https://github.com/mcarans/flakeheaven/blob/master/docs/config.md#plugins), not everything installed.
++ [Make output beautiful](https://github.com/mcarans/flakeheaven/blob/master/docs/formatters.md).
 + [pyproject.toml](https://www.python.org/dev/peps/pep-0518/) support.
-+ [Check that all required plugins are installed](https://flakehell.readthedocs.io/commands/missed.html).
-+ [Syntax highlighting in messages and code snippets](https://flakehell.readthedocs.io/formatters.html#colored-with-source-code).
++ [Check that all required plugins are installed](https://github.com/mcarans/flakeheaven/blob/master/docs/commands/missed.md).
++ [Syntax highlighting in messages and code snippets](https://github.com/mcarans/flakeheaven/blob/master/docs/formatters.md#colored-with-source-code).
 + [PyLint](https://github.com/PyCQA/pylint) integration.
-+ [Powerful GitLab support](https://flakehell.readthedocs.io/formatters.html#gitlab).
++ [Powerful GitLab support](https://github.com/mcarans/flakeheaven/blob/master/docs/formatters.md#gitlab).
 + Codes management:
     + Manage codes per plugin.
     + Enable and disable plugins and codes by wildcard.
-    + [Show codes for installed plugins](https://flakehell.readthedocs.io/commands/plugins.html).
-    + [Show all messages and codes for a plugin](https://flakehell.readthedocs.io/commands/codes.html).
+    + [Show codes for installed plugins](https://github.com/mcarans/flakeheaven/blob/master/docs/commands/plugins.md).
+    + [Show all messages and codes for a plugin](https://github.com/mcarans/flakeheaven/blob/master/docs/commands/codes.md).
```

Comment:
all the above links should probably point to `https://github.com/flakeheaven/flakeheaven` I think instead of `https://github.com/mcarans/flakeheaven`